### DISCUSSION TITLE
How to set the language of ESP32-S3-BOX

### DISCUSSION THREAD

**Discussion by xqyang2008** [toverainc/willow#383] [Q&A]
I am used to using simplified Chinese, but it seems that the current ESP32-S3-BOX can only display English. If I ask it to display Chinese, garbled characters will appear. So is it possible to configure the firmware language?

**Comment by xqyang2008** [comment]
Neither TTS nor ASR work properly

**Comment by 6PATyCb** [comment]
You can build your own firmware in which you can replace the font file. I did this to support Cyrillic. 
![изображение](https://github.com/user-attachments/assets/be3e22a6-1c09-469a-a591-a7b065eb2b38)

**Comment by 6PATyCb** [comment]
You can see how to create a font file here https://forum.lvgl.io/t/esp32-lvgl-example-with-cyrillic-fonts/9756
